roles/nsx/tasks/main.yml

```diff
@@ -0,0 +1,159 @@
+---
+- name: NSX | Run kube proxy
+  run_once: true
+  import_role:
+    name: kube_proxy_and_dns
+  tags: nsx
```

Comment:
Not sure if tags are necessary here - openshift-ansible doesn't use them. The recommended way to run a part of deploy is to create a new playbook in `playbooks/<subfolder>`

Reply:
fixed with 11e4f05048c54c2d7b2d1f981cb101a0471be058